iroh/src/baomap/flat.rs

```diff
@@ -601,6 +612,16 @@ impl Map for Store {
             None
         }
     }
+
+    fn contains_complete(&self, hash: &Hash) -> bool {
```

Comment:
You think it is worth it to have these two? I was thinking you would just call get for a membership test. Or maybe have a fn that takes a hash and returns an enum Complete/Partial/NotFound?

Reply:
I changed it to an enum. Thought about this too and agree that it makes more sense than to have to methods.